### DISCUSSION TITLE
제페토 월드 최대 접속 인원

### DISCUSSION THREAD

**Discussion by strangestraw** [naverz/zepeto-studio-kor#185] [5. Q&A]
안녕하세요.
제페토 월드 최대 접속 인원을 16명으로 알고 있었는데,
12월 20일 가이드 내용으로는 8명 제한으로 나와있습니다.
20일 이전에는 16명이었던 내용이 8명으로 변경된건가요?
마찬가지로 관전자도 불가하다고 나오는데 변경된 내용인지 궁금합니다.
확인 부탁드립니다.
감사합니다.

**Comment by x4cf** [comment]
1.0 4버전에는 접속인원을 수정할 수 없었습니다. 가이드에는 이전 버전에 대한 안내가 있었던것같아요.

1.1.1버전 이후에서는 2에서 16명까지 자유롭게 설정하시면됩니다 :)

아직 월드에는 관전자 설정이 없습니다

**Reply by strangestraw** [reply]
답변 감사합니다!
